jaseci_ai_kit/jac_nlp/jac_nlp/action_configs/bi_enc_action_config.py

```diff
@@ -46,20 +48,28 @@
                             {
                                 "name": "prod-script",
                                 "configMap": {"name": "bi-enc-up", "defaultMode": 420},
-                            }
+                            },
+                            {
+                                "name": "jac-nlp-volume",
+                                "persistentVolumeClaim": {"claimName": "jac-nlp-pvc"},
+                            },
                         ],
                         "containers": [
                             {
                                 "name": "bi-enc",
-                                "image": "jaseci/jac-nlp:latest",
-                                "command": ["bash", "-c", "source script/prod_up"],
+                                "image": "jaseci/jaseci-experiment:1.4.0.12",
+                                "command": ["bash", "-c", "source /script/prod_up"],
                                 "ports": [{"containerPort": 80, "protocol": "TCP"}],
                                 "resources": {
-                                    "limits": {"memory": "3Gi"},
-                                    "requests": {"memory": "3Gi"},
+                                    "limits": {"memory": "2Gi"},
+                                    "requests": {"memory": "2Gi"},
                                 },
                                 "volumeMounts": [
-                                    {"name": "prod-script", "mountPath": "/script"}
+                                    {"name": "prod-script", "mountPath": "/script"},
```

Comment:
Should also make sure this volumeMounts configuration is there for all the action_config. Not sure if that's the case right now.

Reply:
As well as the volume spec a few lines above this.

```
                            {
                                "name": "jac-nlp-volume",
                                "persistentVolumeClaim": {"claimName": "jac-nlp-pvc"},
                            },
```